test_data/api_calls/ebv_api.http

```diff
@@ -0,0 +1,31 @@
+# @name session
+POST http://localhost:3030/api/login
+Content-Type: application/json
+
+{
+  "email": "admin@localhost",
+  "password": "adminadmin"
+}
+
+###
+
+# @name lastTask
+PUT http://localhost:3030/api/ebv/overviews/dataset_sm.nc
+Authorization: Bearer {{session.response.body.$.id}}
+Content-Type: application/json
+Accept: application/json
+
+###
+
+GET http://localhost:3030/api/tasks/{{lastTask.response.body.$.taskId}}/status
```

Comment:
I get an error

>{
  "status": "failed",
  "error": "NetCdfCf provider cannot create overviews for `dataset_sm.nc`: DatasetIsNotInProviderPath",
  "cleanUp": {
    "status": "completed",
    "info": null
  }
}

Reply:
[fd4357e](https://github.com/geo-engine/geoengine/pull/931/commits/fd4357e04f4c94c427360749a7c215f917a9b667)